vspace/src/main/webapp/WEB-INF/views/staff/modules/edit.html

```diff
@@ -0,0 +1,29 @@
+<html layout:decorate="~{layouts/main_staff}">
+
+<body>
+	<div class="main-content" layout:fragment="content">
+		<h1>Edit Module: [(${moduleForm.name})]</h1>
+
+		<form method="POST"
+			th:action='@{|/staff/module/${moduleId}/edit?${_csrf.parameterName}=${_csrf.token}|}' th:object="${moduleForm}"
+			enctype="multipart/form-data" style="margin-top: 40px">
```

Comment:
since you're not submitting any files, you can just remove the enctype and use the default.